spec/lib/jobs/install-os-job-spec.js

```diff
@@ -152,6 +152,38 @@ describe('Install OS Job', function () {
         );
     });
 
+    describe('test _validateOptions', function() {
```

Comment:
It seems that It also have more previous unittest debts in this function. anyway, let me add them all